client/index.html

```diff
@@ -8,6 +8,6 @@
   </head>
   <body>
     <div id="app"></div>
-    
+    <script type="module" src="client/components/WildLife.jsx"></script>
```

Comment:
this shouldn't be necessary because you're importing the WildLife component in App.jsx. You can remove after this is approved, not a big deal.

Reply:
great catch @mikesamm!